e2e-tests/e2e-fixtures/dateConstants.js

```diff
@@ -1,295 +1,73 @@
-
-const twoYearsAgo = sub(today, { years: 2 });
-const twoYearsAgoUnix = getUnixTime(twoYearsAgo).toString();
-const twoYearsAgoDay = format(twoYearsAgo, longDayFormat);
-const twoYearsAgoMonth = format(twoYearsAgo, longMonthFormat);
-const twoYearsAgoYear = format(twoYearsAgo, longYearFormat);
-
-const threeYearsAgo = sub(today, { years: 3 });
-const threeYearsAgoDay = format(threeYearsAgo, longDayFormat);
-const threeYearsAgoMonth = format(threeYearsAgo, longMonthFormat);
-const threeYearsAgoYear = format(threeYearsAgo, longYearFormat);
-
-const oneYearAgo = sub(today, { years: 1 });
-const oneYearUnix = getUnixTime(oneYearAgo).toString();
-const oneYearAgoDay = format(oneYearAgo, longDayFormat);
-const oneYearAgoMonth = format(oneYearAgo, longMonthFormat);
-const oneYearAgoYear = format(oneYearAgo, longYearFormat);
-
-const threeYears = add(today, { years: 3, months: 3, days: 1 });
-const threeYearsDay = format(threeYears, longDayFormat);
-const threeYearsMonth = format(threeYears, longMonthFormat);
-const threeYearsYear = format(threeYears, longYearFormat);
-
-const threeDaysAgoPlusMonth = add(threeDaysAgo, { months: 3 });
-
-const sixYearsOneDay = add(today, { years: 6, months: 0, days: 1 });
-const sixYearsOneDayDay = format(sixYearsOneDay, longDayFormat);
-const sixYearsOneDayMonth = format(sixYearsOneDay, longMonthFormat);
-const sixYearsOneDayYear = format(sixYearsOneDay, longYearFormat);
-
-const twelveMonthsOneDay = add(today, { months: 12, days: 1 });
-const twelveMonthsOneDayDay = format(twelveMonthsOneDay, longDayFormat);
-const twelveMonthsOneDayMonth = format(twelveMonthsOneDay, longMonthFormat);
-const twelveMonthsOneDayYear = format(twelveMonthsOneDay, longYearFormat);
-
-const twelveMonthsOneDayAgo = sub(today, { months: 12, days: 1 });
-const twelveMonthsOneDayAgoDay = format(twelveMonthsOneDayAgo, longDayFormat);
-const twelveMonthsOneDayAgoMonth = format(twelveMonthsOneDayAgo, longMonthFormat);
-const twelveMonthsOneDayAgoYear = format(twelveMonthsOneDayAgo, longYearFormat);
-
-const todayUnix = getUnixTime(today).toString();
-const todayUnixDay = format(threeDaysAgo, longDayFormat);
-const todayUnixMonth = format(threeDaysAgo, longMonthFormat);
-const todayUnixYear = format(threeDaysAgo, longYearFormat);
-
-const todayFormattedFull = format(today, 'dd MMMM yyyy');
-const todayFormatted = format(today, 'd MMMM yyyy');
-const todayFormattedShort = format(today, 'dd MMM yyyy');
-const tomorrowFormattedFull = format(tomorrow, 'd MMMM yyyy');
-const tomorrowFormattedFacilityPage = format(tomorrow, 'dd MMM yyyy');
-const tomorrowUnix = getUnixTime(tomorrow).toString();
+export const shortDayFormat = 'd';
+export const longDayFormat = 'dd';
+export const shortMonthFormat = 'M';
+export const longMonthFormat = 'MM';
+export const longYearFormat = 'yyyy';
+export const ddMMMyyyyFormat = 'dd MMM yyyy';
+export const dMMMMyyyyFormat = 'd MMMM yyyy';
+export const ddMMMMyyyyFormat = 'dd MMMM yyyy';
```

Comment:
I'd prefer these to all be in uppercase e.g `SHORT_DAY_FORMAT`, `DD_MMM_YYYY_FORMAT`, what do you think?

Reply:
Feels like they'd be easier to read with this when inside individual tests e.g

```ts
const issuedDate = today.DD_MMM_YYYY_FORMAT;
const coverStart = today.DD_MMM_YYYY_FORMAT;
const coverEnd = today.DD_MMM_YYYY_FORMAT;

vs
const issuedDate = today.dMMMMyyyy;
const coverStart = today.dMMMMyyyy;
const coverEnd = today.dMMMMyyyy;
```

Reply:
great idea, thanks!